src/Dialogs/BulkRenamer/RenamerDialog.vala

```diff
@@ -397,14 +395,12 @@ public class Files.RenamerDialog : Granite.Dialog {
         });
 
         if (mod.pos == RenamePosition.PREFIX) {
-            // prefix_box.add (mod_button);
+            prefix_box.append (mod_button);
         } else {
-            // Gtk4: replace with prepend
-            // suffix_box.add (mod_button);
+            suffix_box.append (mod_button);
             // suffix_box.reorder_child (suffix_revealer, -1);
```

Comment:
This actually has a note that it should be `prepend`

Reply:
Well spotted - thanks.